etc/sources.csv

```diff
@@ -1,22 +1,22 @@
 author,year,matchauthoryear,matchtitle,matchtype,matchid
 Shiraliev,1997,shiraliev1997,Azerbajdzhanskij jazyk,,wals:5521
 Mehraliev,p.c. 2017,,,,
-Shiraliev,1971,,,,
+Shiraliev,1971,,,,hh:g:Shiraliev:Azerbajdzanskogo
 Poppe,1964,nicholas1964,Bashkir Manual,grammar,hh:g:Poppe:Bashkir
 Gisbert Fanselow,p.c. 2017,,,,
-Yuldashev,1981,,,,
-Yuldashev,1997,,,,
-Garinov,1959,,,,
+Yuldashev,1981,,,,mpieva:Juldasev1981Grammati
+Yuldashev,1997,,,,wals:3027
+Garipov,1959,,,,mpieva:Garipov1959Baskirsk
 Dmitriev,1948,dmitriev1948,Grammatika bashkirskogo jazyka,grammar,hh:g:Dmitriev:Bashkirskogo
 Juwon,2011,juwon2011,A grammar of Ewen,grammar,hh:g:Juwon:Ewen
-Pakendorf,in preparation,,,,
+Pakendorf,2020,,The North Siberian Turkic Languages,,
+Pakendorf,2020,,Even and the Northern Tungusic Languages,,
 Todaeva,1997,b1997,Baoan'skij Jazyk,grammar_sketch,hh:s:Todaeva:Baoanskij
 Fried,2010,fried2010,"A grammar of Bao'an Tu, a Mongolic language of northwest China",grammar,hh:g:Fried:Baoan-Tu
 Rybatzki,2003,volker2003,Middle Mongol,grammar_sketch,hh:s:Rybatzki:Middle-Mongol
 Hugjiltu,2003,wu2003,Bonan,grammar_sketch,hh:s:Hugjiltu:Bonan
-Hugkiltu,2003,,,,

```

Comment:
It would be better, to map this entry to `hh:s:Hugjiltu:Bonan` as well. This will take care of fixing the references. If rows are simply deleted, looking up the refs we find in the data will fail.